examples/archive.html

```diff
@@ -146,6 +156,8 @@ <h3 id="offcanvasRightLabel">Images</h3>
         let imageURL = event.target.previousElementSibling.src;
         let image = L.distortableImageOverlay(imageURL);
         map.imgGroup.addLayer(image);
+        placedImagesArray.push[image]
+        imageGroup.addLayer(image[i])
```

Comment:
here, i think we can just say `image` and not `image[i]` anymore, since we aren't in a for loop anymore.
```suggestion
        placedImagesCollection.addLayer(image)
```